.github/workflows/cibuildwheel.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+    env:
+      # Seems to have trouble parsing our setup.py, thus:
+      CIBW_PROJECT_REQUIRES_PYTHON: ">=3.7"
+      # Install numpy with a wheel (do NOT compile it):
+      CIBW_BEFORE_BUILD: "pip install --only-binary :all: -r requirements.txt"
+      # There are no numpy wheels for these:
+      CIBW_SKIP: pp37-* pp38-* cp310-win32 cp310-manylinux_i686 *-musllinux_*
+      # Quick test with C code using NumPy C API:
+      CIBW_TEST_COMMAND: "python {package}/Tests/test_Cluster.py"
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Optional
+          submodules: true  # Optional, use if you have submodules
+
+      - name: Checkout tag/commit
+        run: |
+          git submodule update --init biopython
+          cd biopython
+          # This was the commit for Biopython 1.79 before tagged
+          git checkout 048c4cdd717e9eb56affaaf66275b90d7dc274e9
```

Comment:
Yes. Looking back over the old wheel builds, I started with the release tag but often the wheels failed and something needed fixing - so I had to fix the main repo and either edit the tag or use the commit hash. I started using a hash instead *before* creating the tag on the same hash.

If this goes smoothly, then we should be able to tag first, and set the tag here. It could even push the wheels to PyPI automatically...